Command/ExportCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Command;
+
+use EzSystems\RecommendationBundle\Export\Exporter;
+use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExportCommand extends ContainerAwareCommand
+{
+    /** @var \EzSystems\RecommendationBundle\Export\Exporter */
+    private $exporter;
+
+    /** @var \Psr\Log\LoggerInterface */
+    private $logger;
+
+    /**
+     * @param \EzSystems\RecommendationBundle\Export\Exporter $exporter
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function __construct(
+        Exporter $exporter,
+        LoggerInterface $logger
+    ) {
+        parent::__construct();
+
+        $this->exporter = $exporter;
+        $this->logger = $logger;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('ezreco:runexport')
+            ->setDescription('Run export to files.')
+            ->addOption('webHook', null, InputOption::VALUE_REQUIRED, 'Guzzle Client base_uri parameter, will be used to send YC data')
+            ->addOption('transaction', null, InputOption::VALUE_REQUIRED)
+            ->addOption('host', null, InputOption::VALUE_REQUIRED, 'Host used in exportDownload url for notifier in export feature')
+            ->addOption('customerId', null, InputOption::VALUE_OPTIONAL, 'Your YOOCHOOSE customer ID')
+            ->addOption('licenseKey', null, InputOption::VALUE_OPTIONAL, 'Your YOOCHOOSE license key')
+            ->addOption('lang', null, InputOption::VALUE_OPTIONAL, 'List of languages, eg: eng-GB,fre-FR')
```

Comment:
`List of language codes`, because languages can be interpreted as language name.